xtas/tasks/single.py

```diff
@@ -493,7 +493,7 @@ def corenlp(doc, output='raw'):
 
 
 @app.task
-def corenlp_lemmatize(doc, output='raw'):
```

Comment:
This function is identical to corenlp(). I guess it is supposed to do something different?